api/src/main/java/com/thesurvey/api/exception/GlobalAPIExceptionHandler.java

```diff
@@ -12,20 +12,22 @@
 @RestControllerAdvice
 public class GlobalAPIExceptionHandler {
 
-    /*
-        @handleExceptionMapperException handles the exception thrown by ExceptionMapper.
-    */
+    /**
+     * handles exception thrown by ExceptionMapper.
+     */
     @ExceptionHandler(ExceptionMapper.class)
-    public ResponseEntity<String> handleExceptionMapperException(Exception e) {
-        return ResponseEntity.badRequest().body(e.getMessage());
+    public ResponseEntity<String> handleExceptionMapperException(Exception error) {
```

Comment:
Thank you for this modification `e` to `error`.

I've mentioned and emphasized on **code conventions**. Code conventions is actually for cooperation since all other language, country, whatever people could understand beyond time. And it's really good to keep convention with **FULL NAME** because there's no way to misunderstand this.

But there're some agreeable conventions like `btn` for `button`, `tmp` for `temp`, `bg` for `background` like that. And this `e` is kind of one thing that is agreeable convention. And if you see some utils libraries, you'll see some abbreviated version of words like
```typescript
/**
 * Check if input text is empty string.
 *
 * @param {string} s String type text
 * @returns {boolean} Result of checking empty string
 */
export const isEmptyString = (s: string): boolean => {
  return s === '';
};
```
this way.

But anyway, this kind of modification is sooooo much better than any conventions. Very nice!